env_support/cmake/custom.cmake

```diff
@@ -51,6 +51,8 @@ if(NOT LV_CONF_BUILD_DISABLE_THORVG_INTERNAL)
     target_link_libraries(lvgl_thorvg PUBLIC lvgl)
 endif()
 
+set_source_files_properties(${LVGL_ROOT_DIR}/src/others/vg_lite_tvg/vg_lite_tvg.cpp PROPERTIES COMPILE_FLAGS -Wunused-parameter)
```

Comment:
From [Cmake docs](https://cmake.org/cmake/help/latest/command/set_source_files_properties.html):
> By default, source file properties are only visible to targets added in the same directory (CMakeLists.txt). 

So unless I miss something this line has no effect.

Reply:
I tested it and the warnings are correctly reported after adding this line. I guess it means we cannot add file properties to files added from another CMakeLists.txt.

Reply:
Can we add checks to all C++?

Reply:
Okay then.